packit_service/service/models.py

```diff
@@ -141,12 +145,14 @@ def create(
         chroots: List[str],
         identifier: int = None,
         save: bool = True,
+        build_submitted_time: str = None,
     ):
         b = cls()
         b.identifier = identifier  # generate new ID if None
         b.project = project
         b.owner = owner
         b.chroots = chroots
+        b.build_submitted_time = datetime.utcnow()
```

Comment:
so what type is the `build_submitted_time` now?
`str` or `datetime.datetime` ?

Reply:
Right now its a string. I'll change it to an integer timestamp once I figure out how to fix the postgres problem.